src/anomalib/engine/engine.py

```diff
@@ -869,6 +869,8 @@ def export(
         export_root: str | Path | None = None,
         input_size: tuple[int, int] | None = None,
         transform: Transform | None = None,
+        compression_type: CompressionType | None = CompressionType.FP16,
```

Comment:
If it is only for openvino maybe it should be None by default. Also, can you add this parameter to the cli export command as well? Otherwise just create an issue referencing this PR and I can do it once this is merged.

Reply:
But if None by default, how to compress to fp16 by default? it's a desired behavior in OV.
Could you point a right place for CLI changes?

Reply:
Maybe you can add a check to ensure that when the optimisation type is OpenVINO, the `compression_type` is not None?
Actually, based on this line maybe you won't have to touch `cli.py`. This will get added automatically
https://github.com/openvinotoolkit/anomalib/blob/d5f744ff1d1f4eac7e988f92ca6a20336e751700/src/anomalib/cli/cli.py#L232

Reply:
Hmm, this is a good idea :)

That's what I thought about CLI. Thanks.

Reply:
Merging #1803 simplified that a lot ;)